index.html

```diff
@@ -9,21 +9,165 @@
 </head>
 <body>
     <header>
-        <!-- <img class="logo" src="images/image_geometry_1.png" alt="logo"> -->
         <div class="header-img"></div>
         <nav>
             <button class="mobile-menu"><img src="images/ic_menu.svg" alt="Open Navigation"></button>
         </nav>
     </header>
+
     <section class="headline-section">
-        <h1 class="headline">Charlie Giorgania</h1>
-        <p class="para">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
-            If you like what you see and have a project you need coded, don't hesitate and contact me.
-        </p>
-        <div class="button">
-            <button class="Start-collaboration">Start collaboration</button>
+        <div class="container">
+            <h1 class="tittle">Charlie Giorgania</h1>
+            <p class="para">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don't hesitate and contact me.
+            </p>
+            <button class="cta">Start collaboration</button>
         </div>
     </section>
+
     <div class="header-img2"></div>
+
+    <section class="projects">
+
+        <h1 class="tittle t2">Projects</h1>
```

Comment:
- [ OPTIONAL] Your project is perfect, keep up the great work, but there are a few things you will need to change later, I mark this review as optional but I strongly recommend you downgrade this h1 to an `hx` below since you already have an`h1` tag for your hero section, following Html best practice, you should not have more than 1 h1 in your web app.